examples/go/a2a_basic/README.md

```diff
@@ -6,7 +6,7 @@ It sets up a "Root Agent" that delegates tasks to a local "Roll Agent" and a rem
 ## Prerequisites
 
 Before running this example, you need to have the A2A serving example running.
-Follow the instructions in `cmd/a2a_basic/remote_a2a/check_prime_agent/README.md` to start the serving agent on port 8001.
+Follow the instructions in `remote_a2a/check_prime_agent/README.md` to start the serving agent on port 8001.
```

Comment:
remote_a2a/check_prime_agent/README.md

Please make this a link. I'm not sure if this is a docs repo reference or an adk-go repo reference